metadata/src/test/java/org/apache/kafka/controller/ReplicationControlManagerTest.java

```diff
@@ -252,6 +252,73 @@ public void testGlobalTopicAndPartitionMetrics() throws Exception {
         assertEquals(0, ctx.metrics.globalPartitionCount());
     }
 
+    @Test
+    public void testOfflinePartitionAndReplicaImbalanceMetrics() throws Exception {
+        ReplicationControlTestContext ctx = new ReplicationControlTestContext();
+        ReplicationControlManager replicationControl = ctx.replicationControl;
+
+        for (int i = 0; i < 4; i++) {
+            registerBroker(i, ctx);
+            unfenceBroker(i, ctx);
+        }
+
+        CreatableTopicResult foo = ctx.createTestTopic("foo", 
```

Comment:
This is definitely a nitpick, but can you put this one fewer lines? 2 lines should be enough for this. Same for the ones below.